lib/Doctrine/DBAL/Platforms/MariaDb102Platform.php

```diff
@@ -17,19 +17,21 @@
  * <http://www.doctrine-project.org>.
  */
 
+declare(strict_types=1);
+
 namespace Doctrine\DBAL\Platforms;
 
 use Doctrine\DBAL\Types\Type;
 
 /**
  * Provides the behavior, features and SQL dialect of the MariaDB 10.2 (10.2.7 GA) database platform.
  *
- * Note: Should not be used with versions prior ro 10.2.7.
+ * Note: Should not be used with versions prior to 10.2.7.
  *
  * @author Vanvelthem Sébastien
  * @link   www.doctrine-project.org
  */
-final class MariaDb1027Platform extends MySqlPlatform
+class MariaDb102Platform extends MySqlPlatform
```

Comment:
The rename is a BC break, and should be reverted: if you need an additional abstract inheritance level, adding it will be more effective than renaming herr 👍